lib/chef/resource/file.rb

```diff
@@ -115,6 +117,18 @@ def manage_symlink_source(arg=nil)
           :kind_of => [ TrueClass, FalseClass ]
         )
       end
+
+      def verify(command=nil, opts={}, &block)
+        if ! (command.nil? || [String, Symbol].include?(command.class))
+          raise ArgumentError, "verify requires either a string, symbol, or a block"
+        end
+
+        if command || block_given?
+          @user_verifications << Verification.new(self, command, opts, &block)
```

Comment:
I agree with @lamont-granquist  - I don't think the `user_` is any more informative than `file_` would be, so leave it off if there are no naming collisions.